spec/jobs/cron/stalled_declarative_procedures_job_spec.rb

```diff
@@ -0,0 +1,126 @@
+RSpec.describe Cron::StalledDeclarativeProceduresJob, type: :job do
+  describe "perform" do
+    let(:date) { Time.utc(2017, 9, 1, 10, 5, 0) }
+    let(:instruction_date) { date + 120 }
+
+    let(:state) { nil }
+    let(:procedure) { create(:procedure, :published, :for_individual, :with_instructeur, declarative_with_state: state) }
+    let(:nouveau_dossier1) { create(:dossier, :en_construction, :with_individual, :with_attestation, procedure: procedure) }
+    let(:nouveau_dossier2) { create(:dossier, :en_construction, :with_individual, :with_attestation, procedure: procedure) }
+    let(:dossier_recu) { create(:dossier, :en_instruction, :with_individual, procedure: procedure) }
+    let(:dossier_brouillon) { create(:dossier, procedure: procedure) }
+    let(:dossier_repasse_en_construction) { create(:dossier, :en_construction, :with_individual, procedure: procedure) }
+
+    before do
+        Timecop.freeze(date)
+        dossier_repasse_en_construction&.touch(:declarative_triggered_at)
+      end
+
+    subject(:perform_job) do
+    dossiers = [
+      nouveau_dossier1,
+      nouveau_dossier2,
+      dossier_recu,
+      dossier_brouillon,
+      dossier_repasse_en_construction
+    ].compact
+
+    Cron::StalledDeclarativeProceduresJob.new.perform
+
+    dossiers.each(&:reload)
+  end
+
+    after { Timecop.return }
+
+    context "with some dossiers" do
+      context "en_construction" do
+        let(:state) { Dossier.states.fetch(:en_instruction) }
+        let(:last_operation) { nouveau_dossier1.dossier_operation_logs.last }
+
+        it {
+          perform_job
+          expect(nouveau_dossier1.en_instruction?).to be_truthy
```

Comment:
juste pour info, t'embête pas à changer mais on doit pouvoir faire

```suggestion
          expect(nouveau_dossier1).to be_en_instruction??
```